cdk-eventbridge-pipes-sqs-to-stepfunctions-filter-transform-python/example-pattern.json

```diff
@@ -0,0 +1,54 @@
+{
+  "title": "EventBridge Pipe from SQS to Step Functions with filter and transformation",
+  "description": "Amazon EventBridge Pipe that filters and transforms messages from Amazon SQS to AWS Step Functions.",
+  "language": "Python",
+  "level": "300",
+  "framework": "CDK",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "The example will create an Amazon SQS queue, a AWS Step Functions workflow and an Amazon EventBridge Pipe.",
+      "Sending messages to the Amazon Simple Queue Service will trigger the pipe to start the AWS Step Function workflow execution.",
+      "SQS Messages will be filtered and transformed before sending to AWS Step Functions."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/cdk-eventbridge-pipes-sqs-to-stepfunctions-filter-transform-python",
+      "templateURL": "serverless-patterns/cdk-eventbridge-pipes-sqs-to-stepfunctions-filter-transform-python",
+      "projectFolder": "eventbridge_pipes_sqs_to_stepfunctions_python",
+      "templateFile": "eventbridge_pipes_sqs_to_stepfunctions_python_stack.py"
```

Comment:
Thanks for making the changes, please also adjust these references and make sure that the `templateFile` includes the folder name.